validate.go

```diff
@@ -72,18 +73,23 @@ var bundleValidateCommand = cli.Command{
 			logrus.Fatalf("Rootfs: %v is not a directory.", spec.Root.Path)
 		}
 
-		bundleValidate(spec, rootfsPath)
+		hooksCheck := context.Bool("hooks-check")
+		bundleValidate(spec, rootfsPath, hooksCheck)
 		logrus.Infof("Bundle validation succeeded.")
 	},
 }
 
-func bundleValidate(spec rspec.Spec, rootfs string) {
+func bundleValidate(spec rspec.Spec, rootfs string, hooksCheck bool) {
 	checkMandatoryField(spec)
 	checkSemVer(spec.Version)
 	checkPlatform(spec.Platform)
 	checkProcess(spec.Process, rootfs)
 	checkMounts(spec.Mounts, rootfs)
 	checkLinux(spec.Linux, rootfs)
+
+	if hooksCheck {
+		checkHooks(spec.Hooks)
```

Comment:
how about use checkHooks(spec.Hooks, hookscheck).
Since absolute path should be checked anyway.